### PR TITLE
PS-4497 : Incorrect option error message for mysqlbinlog

### PR DESCRIPTION
**Pull request by nitendra-bhosle**
Fix :  Changed error message to display --force-read instead of --force
       in log_event.cc file

### CODE REVIEW

File: sql/log_event.cc [hunk 1]
```diff
@@ -1586,7 +1586,7 @@ Log_event* Log_event::read_log_event(const char* buf, uint event_len,
       if (!force_opt)
       {
         *error= "No point in reading encrypted binlog - quitting. "
-                "Start mysqlbinlog with --force if you want to attempt "
+                "Start mysqlbinlog with --force-read if you want to attempt "
                 "to read encrypted binlog without decryption.";
```
Comment (LenzGr) [inline sql/log_event.cc]: Not related to the issue itself, but just spotted this:
```suggestion
                "to read an encrypted binlog without decryption.";
```

**Comment by nitendra-bhosle** [comment]
Jenkins Run : https://ps57.cd.percona.com/job/percona-server-5.7-param/418/

File: sql/log_event.cc (hunk 1 above)
Comment (nitendra-bhosle) [reply sql/log_event.cc]: Addressed this change

**Review by venkatesh-prasad-v — APPROVED** [review]
Hi Nitendra! Thank you for working on this issue. Patch looks good and is approved from my side.

**Pull request merged**